bundle.Dockerfile

```diff
@@ -12,18 +12,5 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v2
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 
```

Comment:
@periklis ran target you suggested and got these other changes too.  Is this what you would expect?

/hold

Reply:
This should stay. Unfortunately this is a hindsight of the generation process.

Reply:
"stay" as in I need to manually add it back in?